app/data_providers/translation.py

```diff
@@ -0,0 +1,59 @@
+import json
+from abc import ABC, abstractmethod
+from functools import lru_cache
+from os import path
+from typing import Dict, Any
+
+from app.models.localemodel import LocaleModel
+
+
+class TranslationProvider:
+    def __init__(self, translations_dir: str):
+        self.translations_dir = translations_dir
+
+    @lru_cache(maxsize=None)
```

Comment:
lru cache?? do you load many translations in real-time, so that they don't fit in memory and you need to keep only the most recently used ones?  it seems not

Reply:
`@cache` was added in Python 3.9. My understanding is that `@lru_cache(maxsize=None)` is the way to achieve a simple unbounded cache in Python 3.8. I'll add a comment to that effect.

Reply:
I just don't understand why you need any cache here at all